### DISCUSSION TITLE
What features should be in v5.0?

### DISCUSSION THREAD

**Discussion by clange** [IndustrialDataSpace/InformationModel#547] [Ideas]
We have [a milestone for v5.0 with numerous issues](https://github.com/International-Data-Spaces-Association/InformationModel/milestone/7), however: what are the features that you = the community really need?
* We also need a minor release v4.2, which includes [as many non-breaking changes as possible](https://github.com/International-Data-Spaces-Association/InformationModel/compare/main...develop). This is required for current data space implementation efforts in the mobility domain and other domains. This may require a hackathon of the development team.
* Faster release cycles are helpful to avoid getting stuck as above – let's not be afraid of multiple minor releases
* Certain RAM 4.0 principles, e.g., streaming (data plane vs. control plane), and further implications of the RAM 4.0 such as IDS-REST and the implications of the latter on messages, policies, etc., will have an impact on [the information layer, which itself is being consolidated](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/projects/10), and should not be covered in the information layer's implementation = the infomodel too late in the future.
* Further beyond, think of topics that came from Gaia-X's identity and trust mechanisms such as Verifiable Credentials / Verifiable Presentations / Decentralized Identifiers (cf. [the IDSA ThinkTank repo](https://github.com/International-Data-Spaces-Association/IDS-ThinkTank), accessible for IDSA members).

**Comment by JohannesLipp** [comment]
Please see #550 for the minor release v4.2.0
